popde/adaptive_kde.py

```diff
@@ -291,6 +296,21 @@ def loo_cv_score(self, rescale_factors_alpha):
             float: Negative sum of log likelihood over test data.
         """
         rescale_val = rescale_factors_alpha[:-1]
+        # Ensure rescale_val is a numpy array
+        rescale_val = np.array(rescale_val)
+
+        # Apply shared bandwidth 
+        if self.uniform_rescale:
+            assert all(item in self.dim_names for item in self.uniform_dims), "Some elements in cut_seq are not in seq"
```

Comment:
I don't understand this error message, 'cut_seq' and 'seq' are not related to our code